themes/bootstrap3/js/common.js

```diff
@@ -394,7 +437,10 @@ var VuFind = (function VuFind() {
     getCurrentSearchId: getCurrentSearchId,
     setCurrentSearchId: setCurrentSearchId,
     initResultScripts: initResultScripts,
-    setupQRCodeLinks: setupQRCodeLinks
+    setupQRCodeLinks: setupQRCodeLinks,
+    setInnerHtml,
```

Comment:
I assume you're taking advantage here of the fact that `setInnerHtml` is equivalent to `setInnerHtml: setInnerHtml`. If that's the case, I think it may be confusing to only use that syntax for the last three things in this list. Might be better to either use the more verbose syntax for the new entries, or convert everything to the concise syntax, in the interest of consistency.

Reply:
Oops, didn't mean to do that. Fixed!